src/routes/chores.routes.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from "express";
+import { getChores, createChore, updateChoreStatus } from "../controllers/chores.controller.js";
+
+const router = Router();
+
+router.route("/").get(getChores);
```

Comment:
add `verifyJwt` (from `src/middlware/auth.middleware.ts`)

```suggestion
router.route("/").get(verifyJwt, getChores);
```